### ISSUE TITLE
Theme Selection

### ISSUE THREAD

**Issue by nteinert2005** [briancodex/community-website#14]
Hey Brian,

I would love to write some code that would allow users to be able to switch colors and themes. 

Is this a viable option?

**Comment by briancodex** [comment]
yeah, that's a good idea! do you want to wait until everyone submits their section first, so you can see their design? or are you planning on creating a general theme file that works regardless of what people create?

**Comment by abhinavmaharana** [comment]
Can I design the ui in Adobe XD for homepage if I get a clear what is this site all about

**Comment by briancodex** [comment]
@abhinavmaharana there's not a specific design or theme that we have right now. I'm just letting people create whatever design they want, but eventually, once everyone submits their first portion of their code, I can create a new rubric to refactor specific sections

**Comment by nteinert2005** [comment]
Yeah I was thinking a generic theme file to handle colors, fonts, etc. I already have part of it drafted.

**Comment by briancodex** [comment]
you could try creating one now and submitting a PR, then once more people add their code I can create a new issue where someone goes in and implements the styles and colors you created.

**Comment by nteinert2005** [comment]
Thanks. I will do a PR later today

**Comment by nteinert2005** [comment]
@briancodex need access to be able to push this update. 

I did `git branch theming && git push origin theming` and got a 404 with permission issues.

**Comment by nteinert2005** [comment]
@briancodex do I have permission to do a PR now?

**Comment by abhinavmaharana** [comment]
bro try to use git push origin HEAD:theming
it will work in my case it worked

**Comment by briancodex** [comment]
@nteinert2005 you don't need any permissions from me to push. You should just push like you normally do when adding to github
